Source/WebGPU/WebGPU/RenderBundleEncoder.mm

```diff
@@ -1156,12 +1166,6 @@ - (instancetype)initWithICB:(id<MTLIndirectCommandBuffer>)icb containerBuffer:(i
             return;
         }
 
-        auto sum = checkedSum<uint64_t>(offset, size);
```

Comment:
this was not just redundant with the logic in `computedSizeOverflows` but incorrect since `size == WGPU_WHOLE_SIZE` was not considered